domains/securesandbox.json

```diff
@@ -0,0 +1,31 @@
+{
+  "description": "Dominio para pruebas de cyberseguridad con dominio is-a.dev",
+  "owner": {
+    "username": "CrowleyCrowd",
+    "email": "ealba.platforms@outlook.com",
+    "discord": "crowleycrowd"
+  },
+  "record": {
+    "MX": [
+      {
+        "host/name": "send",
+        "target": "feedback-smtp.us-east-1.amazonses.com",
+        "priority": 10
+      } 
+    ],
+    "TXT": [
+      {
+        "host/name": "send",
+        "target": "v=spf1 include:amazonses.com ~all"
+      },
+      {
+        "host/name": "resend._domainkey",
+        "target": "p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCqstJSWbINtFx1fSomoSdFnVgLNt9dQgSrwrBQKyWqVd/FCUJxSKftjXAUthTCddxGwggtlrm3nbj8/fwNUpTDJ0znmG3JnuSmZ3Ac/HBrnySZjyQw152/A9eW+wMYMsk4DjACYfkGubdIx12i5LS4YFeBbYjtAVHkf4Oy7Xj2EQIDAQAB"
+      },
+      {
+        "host/name": "_dmarc",
+        "target": "v=DMARC1; p=none;"
+      }
+    ]
+  }
```

Comment:
Improperly done records. You need to follow the actual schema. See docs: https://docs.is-a.dev/